MetaMask/metamask-extension
# upgrade gulp-autoprefixer

## Explanation
On every new process we are getting spammed with a message about caniuse browserlists being outdated. This PR upgrades the dependency that depends on that package, and removes the resolution for caniuse-browserslist which was pinned to an older version. In addition the autoprefixer output now caught two issues with us using 'end' and 'start' intsead of 'flex-end' and 'flex-start'. I also added a .browserslistrc file with our target browsers in it which autoprefixer will pickup on and use. 

## Screenshots/Screencaps



### Before
![Screenshot from 2022-11-09 16-51-54](https://user-images.githubusercontent.com/4448075/200959603-2c9d6a6e-ce0c-495d-a0b2-789e745ed780.png)
### After

<details>
<summary>Example of *helpful errors*</summary>

![Screenshot from 2022-11-09 16-56-14](https://user-images.githubusercontent.com/4448075/200960724-54e78ae6-201d-411c-8add-236c10fe2e93.png)


</details>

<details>
<summary>After cleanup</summary>

![Screenshot from 2022-11-09 17-01-38](https://user-images.githubusercontent.com/4448075/200960768-f8363f6b-37ba-499a-95dc-c775fed15c4b.png)



</details>



## Manual Testing Steps
1. pull this branch
2. yarn setup
3. yarn start, notice no spamming of the caniuse-lite is outdated message 
4. profit

## Pre-merge author checklist

  - [x/ How reviewers can test my changes

## Pre-merge reviewer checklist
